src/Site_Docker.php

```diff
@@ -60,6 +60,7 @@ public function generate_docker_compose_yml( array $filters = [] ) {
 				array( 'name' => 'WORDPRESS_DB_PASSWORD' ),
 				array( 'name' => 'USER_ID=${USER_ID}' ),
```

Comment:
Since we're not changing these values, we can just put `USER_ID` instead of `USER_ID=${USER_ID}`